lib/portus/background/registry.rb

```diff
@@ -17,6 +17,14 @@ def work?
         true
       end
 
+      def enabled?
+        if APP_CONFIG("background.registry") == true
+          Rails.logger.warn("WARNING: Registry is disabled!")
+        else
+          APP_CONFIG.enabled?("background.registry")
+        end
+      end
```

Comment:
I don't think `APP_CONFIG("background.registry")` will work. Do this instead:

```ruby
def enabled?
  val = APP_CONFIG.enabled?("background.registry")
  Rails.logger.warn("WARNING: Registry is disabled!") unless val
  val
end
```

After that, notice the following:

1. You don't have to start the message with `WARNING`, since the `warn` method will already cover that.
2. The message might not be clear to users. Instead, prefer something like: "Registry integration has been disabled. This is highly discouraged!". For the `sync`: "Registry synchronization has been disabled. This can lead into consistency problems!"